src/Frontend/Fragment/WhatListPage.hs

```diff
@@ -0,0 +1,43 @@
+{-# LANGUAGE DataKinds         #-}
+{-# LANGUAGE DeriveGeneric     #-}
+{-# LANGUAGE TemplateHaskell   #-}
+
+module Frontend.Fragment.WhatListPage
```

Comment:
we can keep this "fragment", but i wouldn't have started this module: it doesn't have a `ToHtml` instance, and it's very tightly related to the idea lists module it is coming from.